public/javascripts/cohortController.js

```diff
@@ -14,3 +14,12 @@ var cohortController = function($scope,  $http, $state) {
     });
   };
 };
+
+var cohortStudentsController = function($scope, $http, $state) {
+
+  $scope.showStudent = function(student) {
+    $scope.$parent.$parent.currentStudent = student;
```

Comment:
Doesn't just doing scope.currentStudent create a new variable in the child
scope?

On Fri, Apr 29, 2016 at 1:53 PM Nora Mohamed <notifications@github.com>
wrote:

> In public/javascripts/cohortController.js
> <https://github.com/davidabrahams/Spring-Initiative/pull/161#discussion_r61619089>
> :
>
> > @@ -14,3 +14,12 @@ var cohortController = function($scope,  $http, $state) {
> >      });
> >    };
> >  };
> > +
> > +var cohortStudentsController = function($scope, $http, $state) {
> > +
> > +  $scope.showStudent = function(student) {
> > +    $scope.$parent.$parent.currentStudent = student;
>
> does this actually work? in the student controllers i've had to just say
> $scope.currentStudent to access it. If you try to switch between students
> does it work?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/davidabrahams/Spring-Initiative/pull/161/files/4564c5995d55d41f88d42c58d73ffcdc14eb369d#r61619089>
>